src/main/scala/mesosphere/marathon/api/v2/Validation.scala

```diff
@@ -91,6 +93,25 @@ object Validation {
     }
   }
 
+  def fetchUriHasSupportedProtocol: Validator[FetchUri] = {
+    lazy val supportedProtocols = Set("http", "https", "ftp", "ftps", "hdfs")
```

Comment:
We tightly couple the capabilities of mesos to this validation here. 
As much as I like to fail fast, this could also be a limitation.